draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -370,8 +388,15 @@
                 offerer MUST be prepared to receive a DTLS ClientHello message (if a new DTLS association
                 is established by the answerer) from the answerer before the offerer receives the SDP answer.
             </t>
+            <t>
+                NOTE: If the offerer receives a DTLS ClientHello message, and a DTLS association is established,
+                before the offerer receives the SDP Answer carrying the fingerprint associated with the DTLS
+                association, any data received on the DTLS association before the fingerprint is received is to be
+                considered coming from an unverified source. The processing of such data is outside the scope of

```

Comment:
My suggestion is to just say "must not send data", and not start looking into specific protocols etc. Implementations will handle it.